cli/lib/gather.js

```diff
@@ -259,30 +263,48 @@
 					}
 				}
 
-				if (LAUNCH_IMAGE_REGEXP.test(info.name)) {
+				if (this.platform === 'ios' && LAUNCH_IMAGE_REGEXP.test(info.name)) {
 					results.launchImages.set(relPath, info);
 					return;
 				}
 				// }
 				// fall through to lump with JPG...
 
 			case 'jpg':
-				// if the image is the LaunchLogo.png, then let that pass so we can use it
-				// in the LaunchScreen.storyboard
-				const m = info.name.match(LAUNCH_LOGO_REGEXP);
-				if (m) {
-					info.scale = m[1];
-					info.device = m[2];
-					results.launchLogos.set(relPath, info);
 
-				// if we are using app thinning, then don't copy the image, instead mark the
-				// image to be injected into the asset catalog. Also, exclude images that are
-				// managed by their bundles.
-				} else if (this.useAppThinning && !relPath.match(BUNDLE_FILE_REGEXP)) {
-					results.imageAssets.set(relPath, info);
-				} else {
-					results.resourcesToCopy.set(relPath, info);
+				// Toss Android splash screens into launchImages
+				if (this.platform === 'android') {
+					if (info.name === 'default.9' || info.name === 'default') {
+						results.launchImages.set(relPath, info);
+						return;
+					}
+					if (relPath.match(DRAWABLE_REGEXP)) {
+						// Toss Android drawables into imageAssets to be processed via ProcessDrawablesTask
+						results.imageAssets.set(relPath, info);
+						return;
+					} 
```

Comment:
<!--
  1 failure:  cli/lib/gather.js...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: cli/lib/gather.js;
  Line: 285;
-->
- :no_entry_sign: cli/lib/gather.js line 285 – Trailing spaces not allowed. (no-trailing-spaces)